projects/common/src/time/relative-time-range.ts

```diff
@@ -8,6 +8,9 @@ export class RelativeTimeRange implements TimeRange {
   public constructor(public readonly duration: Readonly<TimeDuration>) {
     this.endTime.setTime(Date.now());
     this.startTime.setTime(this.endTime.getTime() - this.duration.toMillis());
+
+    this.endTime.setSeconds(0, 0);
+    this.startTime.setSeconds(0, 0);
```

Comment:
If it's fine, can we maintain this under a boolean and have it true by default for now. The reason being that, this is something very explicit and will help a lot when it comes to understanding the code in the future.
Also, maybe mentioning a comment would help in the future for anyone to know the context too.

I feel this will also affect other places apart from the top bar where time related stuff is used, but this shouldn't be an issue I guess, do test it thoroughly if possible though.

Reply:
Added a boolean with a default value of `true`